pkg/kn/commands/service/configuration_edit_flags.go

```diff
@@ -479,6 +486,32 @@ func (p *ConfigurationEditFlags) Apply(
 		service.Spec.Template.Spec.TimeoutSeconds = &p.TimeoutSeconds
 	}
 
+	if cmd.Flags().Changed("profile") {
+		if len(knconfig.GlobalConfig.Profile(p.Profile).Annotations) > 0 {
+			annotations := knconfig.GlobalConfig.Profile(p.Profile).Annotations
```

Comment:
Per initial description Roland aimed to have built-in `istio` profile as a well known option. @rhuss I wonder if it is still the case. IMO it's valuable to have that specific default at least.